t3.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: t
```

Comment:
Obviously, these are just for testing and will need to be replaced by something with a better name. 😄

Reply:
I am channelling my inner gopher.

Reply:
Yeah, these guys were just for quick testing